src/ui/dialog-vkbasalt.ui

```diff
@@ -0,0 +1,511 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!-- Created with Cambalache 0.10.2 -->
+<interface>
+  <!-- interface-name dialog-vkbasalt.ui -->
+  <requires lib="gtk" version="4.0"/>
+  <requires lib="libadwaita" version="1.0"/>
+  <template class="vkBasaltDialog" parent="AdwWindow">
+    <property name="default-width">550</property>
+    <property name="deletable">False</property>
+    <property name="modal">True</property>
+    <property name="title" translatable="yes">vkBasalt Settings</property>
+    <child>
+      <object class="GtkBox">
+        <property name="orientation">vertical</property>
+        <child>
+          <object class="AdwHeaderBar">
+            <child type="start">
+              <object class="GtkButton" id="btn_cancel">
+                <property name="label" translatable="yes">Cancel</property>
+              </object>
+            </child>
+            <child type="end">
+              <object class="GtkButton" id="btn_save">
+                <property name="label" translatable="yes">Save</property>
+                <style>
+                  <class name="suggested-action"/>
+                </style>
+              </object>
+            </child>
+          </object>
+        </child>
+        <child>
+          <object class="AdwPreferencesPage">
+            <child>
+              <object class="AdwPreferencesGroup">
+                <property name="description" translatable="yes">Configure vkBasalt settings.</property>
```

Comment:
This should be `title`.

Reply:
is this needed at all? the window title already says "vkbasalt settings", so this text doesn't provide value